lib/pf/Switch/Cisco/Catalyst_2960.pm

```diff
@@ -513,9 +513,10 @@ sub returnRadiusAccessAccept {
     if ( isenabled($self->{_UrlMap}) && $self->externalPortalEnforcement ) {
         if( defined($args->{'user_role'}) && $args->{'user_role'} ne "" && defined($self->getUrlByName($args->{'user_role'}))){
             my $mac = $args->{'mac'};
+            $args->{'session_id'} = "sid".$self->setSession($args);
             my $redirect_url = $self->getUrlByName($args->{'user_role'});
-            $args->{'session_id'} = "sid".$self->setSession($args) if ($redirect_url =~ /\$session_id/);
-            $redirect_url =~ s/\$([a-zA-Z_0-9]+)/$args->{$1} \/\/ ''/ge;
+            $redirect_url .= '/' unless $redirect_url =~ m(\/$);
+            $redirect_url .= $args->{'session_id'};
```

Comment:
Ok, then don't forget to change the doc (PacketFence_Network_Devices_Configuration_Guide.asciidoc)